client/reducers/accountInfo.js

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+
+// ACTION TYPES
+const GET_USER_INFO = 'GET_USER_INFO';
+const UPDATE_USER_INFO = 'UPDATE_USER_INFO';
+const DELETE_USER = 'DELETE_USER';
+
+// ACTION CREATORS
+export const getUserInfo = userInfo => {
+  return {type: GET_USER_INFO, userInfo};
+}
+export const updateUserInfo = updatedUser => {
+  return {type: UPDATE_USER_INFO, updatedUser}
+}
+
+export const deleteUser = user => {
```

Comment:
dont need parameter  since you are just resetting to empty obj

Reply:
Done